src/Microsoft.Dnx.Tooling/ConsoleCommands/PublishConsoleCommand.cs

```diff
@@ -51,12 +64,14 @@ public static void Register(CommandLineApplication cmdApp, ReportsFactory report
                         WwwRootOut = optionWwwRootOut.Value() ?? optionWwwRoot.Value(),
                         NoSource = optionNoSource.HasValue(),
                         Runtimes = optionRuntime.HasValue() ?
-                            string.Join(";", optionRuntime.Values).
-                                Split(new[] { ';' }, StringSplitOptions.RemoveEmptyEntries) :
+                            string.Join(";", optionRuntime.Values).Split(new[] { ';' }, StringSplitOptions.RemoveEmptyEntries) :
+                            new string[0],
+                        Frameworks = optionFramework.HasValue() ?
+                            string.Join(";", optionFramework.Values).Split(new[] { ';' }, StringSplitOptions.RemoveEmptyEntries) :
```

Comment:
Seems ripe for a helper method :). I expect we'll be adding another one later for runtime ID filters.